how-to.md

```diff
@@ -0,0 +1,194 @@
+# How to WP-CLI
+
+## Introduction
+
+Using WP-CLI you can perform specific tasks using the command line instead of accessing the WordPress admin panel. In the following section we will cover step by step some of these
+tasks.
+
+## How to install WordPress
+
+Download and install WordPress using WP-CLI is very easy. It's done in four simple steps.
```

Comment:
Try to avoid language that can come off as condescending, like "easy" and "simple", etc...

While I understand what you want to convey with that, it can offend people who do have trouble following what you are writing about. Imagine having trouble getting something to work, and you're constantly being told how "easy" it really is, and then anyone could do it ... "just not you!".